docs/sources/configuration/_index.md

```diff
@@ -389,10 +389,20 @@ grpc_tls_config:
 # CLI flag: -server.log-source-ips-regex
 [log_source_ips_regex: <string> | default = ""]
 
-# Optionally log requests at info level instead of debug level.
+# Optionally log request headers.
+# CLI flag: -server.log-request-headers
+[log_request_headers: <boolean> | default = false]
+
+# Optionally log requests at info level instead of debug level. Applies to
+# request headers as well if server.log-request-headers is enabled.
 # CLI flag: -server.log-request-at-info-level-enabled
 [log_request_at_info_level_enabled: <boolean> | default = false]
 
+# Comma separated list of headers to exclude from loggin. Only used if
```

Comment:
Should "loggin" be "logging"?

Reply:
agreed. But we can't really fix this on Loki :cry: this was autogenerated from a dependency. To fix it we'll have to make the change on dskit and then revendor it.

Reply:
Ah, ok in that case never mind.